### PR TITLE
Add contribution docs

### PR DESCRIPTION
**Pull request by contolini**
## Additions

- `CONTRIBUTING.md` with steps on how to edit CF.

## Review

- @Scotchester 
- @jimmynotjim 

## Notes

- I removed the instruction to increment components' manifests whenever you edit their source code. That responsibility is now listed in the "Release management" section.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+After changes have been made to components, nothing will be published to npm until
+those changes enter the `master` branch.
+
+1. Go to wherever cf is cloned locally: `cd capital-framework`
+1. Ensure you're on canary: `git checkout canary`
+1. Ensure you're up-to-date: `git pull upstream canary`
+1. Open `/CHANGELOG.md` and check the "Unreleased" section. For each component
+that's been modified, open its `/src/COMPONENT/package.json` and increment its
+version. Follow [SemVer](http://semver.org/) and increment either its major, minor,
+or patch version dependent upon the changes that were made to the component.
```
Comment (Scotchester) [inline CONTRIBUTING.md]: So we _don't_ want folks to increment component versions prior to submitting a component update to `canary`?

Comment (contolini) [inline CONTRIBUTING.md]: Yeah, the background on that decision is [here](https://github.com/cfpb/capital-framework/pull/259#issuecomment-170963537).

I guess the discussion is still ongoing. We should wrap up that PR before merging this one.

Comment (Scotchester) [inline CONTRIBUTING.md]: Ah right, thanks for the reminder.

**Comment by jimmynotjim** [comment]
Looks right to me, now I gotta go fix my PR.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Use the issue tracker to suggest feature requests, report bugs, and ask questions. 
+This is also a great way to connect with the developers of the project as well
+as others who are interested in this solution.  
+
+Use the issue tracker to find ways to contribute. Find a bug or a feature, mention in
+the issue that you will take on that effort, then follow the _Changing the codebase_ 
+guidance below.
+
+
+## Changing the codebase
```
Comment (jimmynotjim) [inline CONTRIBUTING.md]: Are we still forking `master` for important bug fixes? Or should we keep the canary process? (FWIW, I think keeping one set of instructions makes things simpler).

Comment (contolini) [inline CONTRIBUTING.md]: Cool, I'll leave it how it is to avoid overwhelming people. l337 users can use master if they want.

**Comment by contolini** [comment]
Rebased. Should be good-to-go.

**Comment by Scotchester** [comment]
Whew, finally! ("Finally, Travis", not "finally, @contolini".)

**Pull request merged**